keps/sig-apps/3939-consider-terminating-pods-jobs/README.md

```diff
@@ -240,10 +251,15 @@ See [Kueue: Account for terminating pods when doing preemption](https://github.c
 
 ### API Name Choices
 
-- TerminatingAsActive
-- ActiveUntilTerminal
-- DelayPodRecreationUntilTerminal
-- ?
+<<[UNRESOLVED  Name for the API]>>
+Options for the API
+
+- TerminatingAsActive (BoolPtr)
+- ActiveUntilTerminal (BoolPtr)
+- DelayPodRecreationUntilTerminal (BoolPtr)
+- recreatePodsWhen (enum of TerminatingOrFailed|Failed)
```

Comment:
I prefer this

Reply:
I suggest you go with this for the KEP, and we can always adjust during code reviews.

Reply:
I forgot to remove this but yea I added this as the api defintion.